src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.test.tsx

```diff
@@ -44,30 +46,56 @@ jest.mock('./track-panel-items/TrackPanelRegularItem', () => () => (
 ));
 
 const mockState = createMockBrowserState();
+const activeGenomeId = mockState.browser.browserGeneral.activeGenomeId;
```

Comment:
Think we can also use the selector to get the `activeGenomeId` by passing the `mockState to it

Reply:
But we don't yet have store.getState at this point. The selector will be upset, because mockState does not have all the properties of RootState.